jsx/Form.js

```diff
@@ -630,6 +630,73 @@ var DateElement = React.createClass({
   }
 });
 
+/**
+ * Time Component
+ * React wrapper for a <input type="time"> element.
+ */
+var TimeElement = React.createClass({
+
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    label: React.PropTypes.string,
+    value: React.PropTypes.string,
+    id: React.PropTypes.string,
+    disabled: React.PropTypes.bool,
+    required: React.PropTypes.bool,
+    onUserInput: React.PropTypes.func
+  },
+
+  getDefaultProps: function() {
+    return {
+      name: '',
+      label: '',
+      value: '',
+      id: null,
```

Comment:
'id' is of type string but here its default is set to null. would an empty string be more consistent here?

Reply:
I think this is an inconsistency that should be addressed in it's own PR since it is the case for all form elements in `Form.js`

Reply:
once again, i dont think inconsistency matters here as https://github.com/aces/Loris/pull/3559 pushed through a change where the default id='' and not null